docs/encyclopedia/worker-versioning-legacy.mdx

```diff
@@ -0,0 +1,265 @@
+
+:::
+
+Worker Versioning simplifies the process of deploying changes to [Workflow Definitions](/workflows#workflow-definition).
+It does this by letting you define sets of versions that are compatible with each other, and then assigning a Build ID to the code that defines a Worker.
+The Temporal Server uses the Build ID to determine which versions of a Workflow Definition a Worker can process.
+
+We recommend that you read about Workflow Definitions before proceeding, because Workflow Versioning is largely concerned with helping to manage nondeterministic changes to those definitions.
+
+Worker Versioning helps manage nondeterministic changes by providing a convenient way to ensure that [Workers](#worker) with different Workflow and Activity Definitions operating on the same [Task Queue](#task-queue) don't attempt to process [Workflow Tasks](#workflow-task) and [Activity Tasks](#activity-task-execution) that they can't successfully process, according to sets of versions associated with that Task Queue that you've defined.
+
+Accomplish this goal by assigning a Build ID (a free-form string) to the code that defines a Worker, and specifying which Build IDs are compatible with each other by updating the version sets associated with the Task Queue, stored by the Temporal Server.
+
+### When and why you should use Worker Versioning
+
+:::caution
+
+This version of Worker Versioning is deprecated and will go away at some point. Please redirect your attention to [Worker Versioning](/workers#worker-versioning)
+
+:::
+
+The main reason to use this feature is to deploy incompatible changes to short-lived [Workflows](/workflows).
+On Task Queues using this feature, the Workflow starter doesn't have to know about the introduction of new versions.
+
+The new code in the newly deployed Workers executes new [Workflow Executions](/workflows#workflow-execution), while only Workers with an appropriate version process old Workflow Executions.
+
+#### Decommission old Workers
+
+You can decommission old Workers after you archive all open Workflows using their version.
+If you have no need to query closed Workflows, you can decommission them when no open Workflows remain at that version.
+
+For example, if you have a Workflow that completes within a day, a good strategy is to assign a new Build ID to every new Worker build and add it as the new overall default in the version sets.
+
+Because your Workflow completes in a day, you know that you won't need to keep older Workers running for more than a day after you deploy the new version (assuming availability).
+You can apply this technique to longer-lived Workflows too; however, you might need to run multiple Worker versions simultaneously while open Workflows complete.
+
+Version sets have a maximum size limit, which defaults to 100 build IDs across all sets.
```

Comment:
```suggestion
Version sets have a maximum size limit, which defaults to 100 Build IDs across all sets.
```

I capitalized it here for consistency with the capitalization in the next bullet point, but it's possible that this one was right and that one is wrong. In any case, all uses of this term should match.